code/routers/country_page/country.js

```diff
@@ -101,10 +101,10 @@ router.get('/get-country-info', (req, res) => {
     inputs:
       countryName: String
 
-    Returns the list of organizations associated with given country
+    Returns the list of organizationIDs associated with given country
```

Comment:
Please add a TODO here to note that this is not finished and that it will be changed back to organizations

Reply:
Now updated it, so I didn't add a todo.